appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 1.0.{build}
+before_build:
+- cmd: dotnet restore
```

Comment:
Appveyor build was [failed](https://ci.appveyor.com/project/odinserj/hangfire-autofac/build/1.0.51) on this command. Could you fix it?